### PR TITLE
Update Jenkins to 2.190.1, update plugins version

### PR DESCRIPTION
**Pull request by denbekkerpas**

### CODE REVIEW

File: latest-release [hunk 1]
```diff
@@ -0,0 +1 @@
+2.176.3-1
```
Comment (bellackn) [inline latest-release]: this file should not be committed

**Review by bellackn — CHANGES_REQUESTED** [review]
the latest-release file should not be committed

File: latest-release (hunk 1 above)
Comment (tobias-richter) [reply latest-release]: Please add this file to the git ignore

**Review by tobias-richter — CHANGES_REQUESTED** [review]

**Comment by tobias-richter** [comment]
Please change the feature branch name to "feature/JENKINS_VERSION-RELEASE_NUMBER" in this example: `feature/2.190.1-1`.
Also the pull request name should reflect what is inside the PR e.g. `Update Jenkins to 2.190.1, update plugins`

**Review by bellackn — APPROVED** [review]
👍

**Review by tobias-richter — APPROVED** [review]

**Pull request merged**